source/_docs/new-relic.md

```diff
@@ -1,21 +1,90 @@
 ---
-title: New Relic
-description: Learn about the New Relic tool in your Pantheon Website Management Platform dashboard
+title: New Relic Pro
+description: Learn how to enable and use New Relic performance metrics and reports for your Pantheon site.
 categories: [developing]
 tags: [debug]
-keywords: new relic,
+keywords: new relic, new relic pro, performance analysis, enable new relic, what is new relic, add new relic, mysql performance, performance, authenticated users, how to log authenticated users, how to use new relic, using new relic, sql performance
 ---
-[New Relic](http://newrelic.com) offers a wide array of metrics that provide a nearly real-time look into the performance of a web application.
+[New Relic Pro](http://newrelic.com) offers a wide array of metrics that provide a nearly real-time look into the performance of a web application and is provided to all sites on Pantheon for free. Using New Relic not only makes it easy for you to monitor to your performance, but it can also speed up the support process by helping our support team visualize corresponding performance and symptoms.
 
-Using New Relic on Pantheon not only makes it easy for you to monitor to your performance, but it can also speed-up the support process by helping out support team visualize corresponding performance and symptoms.
+## Enable New Relic Pro
+To enable New Relic Pro on your Pantheon site, select the **New Relic** tab on your Site Dashboard, and click **Enable New Relic Pro**. New Relic monitoring is automatically enabled for all application servers added to the site, including Multidev environments.
 
-##Enabling New Relic
-Enable New Relic on your Pantheon site by clicking **Settings** in the upper-right corner of your Site Dashboard. Go to the Add Ons tab, and click **Add** next to New Relic.
-![New Relic Add On](/source/docs/assets/images/new-relic-add-on-image.png)
+You should visit your site in the browser a couple of times to generate data in New Relic. After a few minutes pass, go to the New Relic workspace on your Dashboard, and click **Open New Relic**.
 
-For two weeks, you'll receive a trial of New Relic's Pro plan for the site. New relic monitoring is automatically enabled for all application servers added to the site, including Multidev environments.
+## Browser Overview
```

Comment:
I don't think any of this is really necessary. We may be better off just linking to https://docs.newrelic.com/docs/agents/php-agent/getting-started/new-relic-php#monitor-performance for them to dig into what New Relic does.